### PR TITLE
(selenium-chrome-driver) Migrate package to new repository

### LINKED ISSUE #1861: (selenium-chrome-driver) Migrate package
### Discussed in https://github.com/chocolatey-community/chocolatey-packages/discussions/1860

<div type='discussions-op-text'>

<sup>Originally posted by **AdmiringWorm** April 22, 2022</sup>

I was recently made a maintainer of this package, and with that I request it being added to this repository for continued maintenance and to be kept up to date by multiple people.
This ties in to the request of moving the gecko driver, and makes sense to move this driver as well since it is an important package for us and the community.</div>

### PR DESCRIPTION
**Pull request by AdmiringWorm**
## Description

This pull request adds and updates the selenium-gecko-package which I was recently added as a co-maintainer for.
The package has been made an embedded package which we prefer to use when possible, as well as certain metadata items is updated to the correct locations.

## Motivation and Context

The selenium chrome package was severally outdated, and as new maintainer I believed it would make sense to include this package in this repository.

## How Has this Been Tested?

1. Normal installation and uninstallation through the chocolatey-test-environment.
2. Running update.ps1 directly on my local computer

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

## Original Location

N/A

fixes #1861

### CODE REVIEW

**Review by JPRuskin — APPROVED** [review]
Looks good to me.

**Comment by JPRuskin** [comment]
Modified maintainers to enable builds from this repository.

**Comment by AdmiringWorm** [comment]
@JPRuskin I have marked this PR as ready for review again, if you could have another look over it I would appreciate it.

**Review by JPRuskin — APPROVED** [review]
Looks good to me.

File: automatic/selenium-chrome-driver/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  file         = "$toolsDir\"
+  checksum     = '53c2e1f1b9f2a7571483284ea156958674501e8571cfe34be01282b4152e0770138d9aaef12876f109db9d77c1216362d0576aa40b4f19f039d4b97a15054c98'
+  checksumType = 'sha512'
+  destination  = $seleniumDir
+}
+
+Get-ChocolateyUnzip @packageArgs
+Get-ChildItem $toolsDir\*.zip | ForEach-Object { Remove-Item $_ -ea 0 }
+
+Uninstall-BinFile -Name 'chromedriver'
```
Comment (TheCakeIsNaOH) [inline automatic/selenium-chrome-driver/tools/chocolateyInstall.ps1:20]: Should this `Uninstall-Binfile` call be gated behind the parameter like the `Install-BinFile` call is?

Comment (AdmiringWorm) [reply automatic/selenium-chrome-driver/tools/chocolateyInstall.ps1:20]: Not really needed, if it doesn't exist then the call is a no-op (only some debug messages will be outputted).

Comment (TheCakeIsNaOH) [reply automatic/selenium-chrome-driver/tools/chocolateyInstall.ps1:20]: Fair enough.

**Review by TheCakeIsNaOH — APPROVED** [review]

**Review by TheCakeIsNaOH — APPROVED** [review]
The AU script and the package in the test environment are both working for me.

**Comment by TheCakeIsNaOH** [comment]
@AdmiringWorm thanks for getting this added.

**Pull request merged**